react/forms/step3.md

```diff
@@ -0,0 +1,68 @@
+Now we'll modify our form to include a title dropdown.
+
+## Task
+
+1) Make sure the App.js file is still open
+
+2) To create a title dropdown, we'll add a new `value1` attribute to state and a `this.options` array in the constructor.
+
+<pre class="file" data-target="clipboard">
+constructor(props) {
+  super(props);
+  this.state = {
+    value1: 'please choose',
+    value2: '',
+    value3: '',
+    value4: ''
+  };
+  this.onChange = (value1, event) => {
+    this.setState({ value1 });
+  };
+  this.handleTextInputChange1 = value2 => {
+    this.setState({ value2 });
+  };
+  this.handleTextInputChange2 = value3 => {
+    this.setState({ value3 });
+  };
+  this.handleTextInputChange3 = value4 => {
+    this.setState({ value4 });
+  };

```

Comment:
This constructor code already exists. It's not clear that this is a whole sale replacement of all code.

Typically, we only ask users to copy the new code.